nomenclature/codelist.py

```diff
@@ -622,7 +622,7 @@ def validate_units(
             (variable, unit, self.mapping[variable].unit)
             for variable, unit in unit_mapping.items()
             if variable in self.variables
-            and not set(unit) == set(self.mapping[variable].unit)
+            and unit not in to_list(self.mapping[variable].unit)
```

Comment:
The `to_list(self.mapping[variable].unit)` already exists as property `units` of `VariableCode`:
```suggestion
            and unit not in self.mapping[variable].units
```